<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>1072</width>
      <height>617</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabDraw">
     <attribute name="title">
      <string>Draw</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="minimumSize">
         <size>
          <width>770</width>
          <height>570</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>770</width>
          <height>570</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutToolsDraw">
        <item>
         <widget class="QTabWidget" name="tabWidgetDraw">
          <property name="tabShape">
           <enum>QTabWidget::Triangular</enum>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <property name="elideMode">
           <enum>Qt::ElideNone</enum>
          </property>
          <widget class="QWidget" name="tabDrawEllipse">
           <attribute name="title">
            <string>Draw Ellipse</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutDrawEllipse">
              <item>
               <layout class="QVBoxLayout" name="verticalLayoutCenterPoint">
                <item>
                 <layout class="QFormLayout" name="formLayoutCenterPoint">
                  <item row="0" column="0">
                   <widget class="QLabel" name="labelCenterX">
                    <property name="text">
                     <string> x </string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="labelCenterY">
                    <property name="text">
                     <string> y</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QSpinBox" name="spinBoxCenterX">
                    <property name="maximum">
                     <number>770</number>
                    </property>
                    <property name="value">
                     <number>375</number>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QSpinBox" name="spinBoxCenterY">
                    <property name="maximum">
                     <number>570</number>
                    </property>
                    <property name="value">
                     <number>275</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="Line" name="linePoints">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayoutParametrsEllipse">
                <item>
                 <layout class="QFormLayout" name="formLayoutParametrsEllipse">
                  <item row="1" column="0">
                   <widget class="QLabel" name="labelEllipseB">
                    <property name="text">
                     <string> b</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QSpinBox" name="spinBoxEllipseA">
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>770</number>
                    </property>
                    <property name="value">
                     <number>100</number>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QSpinBox" name="spinBoxEllipseB">
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>570</number>
                    </property>
                    <property name="value">
                     <number>100</number>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="labelEllipseA">
                    <property name="text">
                     <string> a </string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonDrawEllipse">
              <property name="text">
               <string>Draw Ellipse</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabDrawSpectrum">
           <attribute name="title">
            <string>Draw Spectrum</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutDrawSpectrum">
              <item>
               <layout class="QVBoxLayout" name="verticalLayoutParametrsSpectrum">
                <item>
                 <layout class="QFormLayout" name="formLayoutDynamicsSpectrum">
                  <item row="0" column="0">
                   <widget class="QLabel" name="labelSpectrumDr">
                    <property name="text">
                     <string>dR</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="labelSpectrumN">
                    <property name="text">
                     <string> N</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QSpinBox" name="spinBoxSpectrumDR">
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>770</number>
                    </property>
                    <property name="value">
                     <number>3</number>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QSpinBox" name="spinBoxSpectrumN">
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>500</number>
                    </property>
                    <property name="value">
                     <number>10</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="Line" name="linePointsInitSpectrum">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayoutInitSpectrum">
                <item>
                 <layout class="QFormLayout" name="formLayoutInitSpectrum">
                  <item row="1" column="0">
                   <widget class="QLabel" name="labelSpectrumB">
                    <property name="text">
                     <string> b </string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QSpinBox" name="spinBoxSpectrumA">
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>770</number>
                    </property>
                    <property name="value">
                     <number>50</number>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QSpinBox" name="spinBoxSpectrumB">
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>570</number>
                    </property>
                    <property name="value">
                     <number>50</number>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="labelSpectrumA">
                    <property name="text">
                     <string> a </string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonDrawSpectrum">
              <property name="text">
               <string>Draw Spectrum</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutColor">
          <item>
           <layout class="QVBoxLayout" name="verticalLayoutColorLabel">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayoutMainColor">
              <item>
               <widget class="QLabel" name="labelTextMainColor">
                <property name="text">
                 <string>Main Color</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelMainColor">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonChangeMainColor">
                <property name="text">
                 <string>Change</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayoutColorBackGround">
              <item>
               <widget class="QLabel" name="labelTextColorBackGround">
                <property name="text">
                 <string>Background</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelColorBackGround">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonChangeColorBackGround">
                <property name="text">
                 <string>Change</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutColorButton">
            <item>
             <widget class="QPushButton" name="pushButtonSwapColor">
              <property name="text">
               <string>Swap</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonClear">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxPainters">
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string/>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tabTimeCircle">
     <attribute name="title">
      <string>Time Cicrle</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutTabTimeCircle"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tabTimeEllipse">
     <attribute name="title">
      <string>Time Ellipse</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutTimeEllipse"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
